scripts/TVs/_utils.py

```diff
@@ -53,9 +53,10 @@ def guess_tv(username):
     if tv is None:
         name = user_utils.get_users_name(username)
         if name is None:
-            utils.print_error(
-                "I don't recognize the username '{}'.  It could be because they don't have an account in the Hackerspace with this username.  You can continue anyway tho.".format(username))
-            return None
+            if utils.confirm(f"I don't recognize the username {username}, it could be because they don't have an account in the Hackerspace with this username. Would you like to continue?"):
```

Comment:
As discussed, please make this more clear.  Would you like to reenter the username?

Reply:
fixed per [c49ab38](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/131/commits/c49ab38060b39c58df77b0b9504c7119ad558ddf)